### PR TITLE
Add testing data to state pages and national map

### PR DESCRIPTION
**Pull request by alegzandach**

### CODE REVIEW

**Review by Lane — CHANGES_REQUESTED** [review]
thanks @alegzandach! this is looking good.  a few things still needed for this:

- the federal page (/federal) should report the number of tests and test rate (if population is available).  right now i'm seeing "unavailable" for both
image

- test rates and spikes for state facilities don't seem to be showing on the state page
- small fix with the console.log in `parseData`

File: packages/gatsby-ucla-site/src/components/states/states.js [hunk 1]
```diff
@@ -303,11 +303,13 @@ export const query = graphql`
             deaths
             deaths_rate
             population
+            tested
```
[the remark this answers is not in the document]
Comment (alegzandach) [reply packages/gatsby-ucla-site/src/components/states/states.js:306]: i had seen this in other metrics with california specifically when i was debugging, is it the expected behavior? unavailable federal data in the table but a blue spike on the map

californiaFederal

[the remark this answers is not in the document]
Comment (alegzandach) [reply packages/gatsby-ucla-site/src/components/states/states.js:306]: oops forgot to check the dev branch, let me make sure this isnt happening there

[the remark this answers is not in the document]
Comment (alegzandach) [reply packages/gatsby-ucla-site/src/components/states/states.js:306]: yes im seeing it there as well

[the remark this answers is not in the document]
Comment (alegzandach) [reply packages/gatsby-ucla-site/src/components/states/states.js:306]: did some digging in california and all 114481 state cases are coming from one facility that has no coords. state facility testing data is being shown in other states, so i think it's just a coincidence?

[the remark this answers is not in the document]
Comment (Lane) [reply packages/gatsby-ucla-site/src/components/states/states.js:305]: ahhh gotcha, thanks for looking in to it!

**Comment by Lane** [comment]
thanks @alegzandach, looks great, merged! 🎉

**Pull request merged**